automation/check-patch.sh

```diff
@@ -205,12 +205,14 @@ run() {
     args+=(
         "provider=$provider"
         "inventory_file=$inventory_file"
+        "cluster=$cluster"
         "ansible_modules_version=$ansible_modules_version"
         "kubevirt_openshift_version=$kubevirt_openshift_version"
         "openshift_playbook_path=$openshift_playbook_path"
         "storage_role=$storage_role"
         "cluster=$cluster"
         "platform=$cluster"
+        "deploy_skydive=true"
```

Comment:
Please move this to a different  PR so we can merge this one without @gbenhaim

Reply:
Done lets wait for the CI